src/components/controls/buttons/DropdownButton.md

```diff
@@ -0,0 +1,51 @@
+
+```
+const alertButtonValues = (event, key) => {
+  alert("Button Key is: " + key)
+}
+
+<DropdownButton 
+  buttonValue='Dropdown'
+  shouldCloseOnButtonClick
+  rootClose>
+  <DropdownButton.Button
+    name="first"
+    styleType="link" 
+    handleOnClick={alertButtonValues}>First Button</DropdownButton.Button>
+  <DropdownButton.Button
+    name="second"
+    styleType="link" 
+    handleOnClick={alertButtonValues}>Second Button</DropdownButton.Button>
+  <DropdownButton.Button
+    name="third"
+    styleType="link" 
+    handleOnClick={alertButtonValues}>Third Button</DropdownButton.Button>
```

Comment:
I think going this route (compound components) puts us in a better position to expand `DropdownButton` to include support for URLs. I think that's something we could address later on as we need it.